README.md

```diff
@@ -14,29 +16,19 @@
 
 [Learn more](https://module-federation.io/)
 
-## 🔥 What does universe offer?
-
-The universe repository provides a set of tools to enhance module federation's capabilities. These include:
+## ✨ What new features does Module Federation 2.0 provide?
 
-- [Module Federation Runtime](https://module-federation.io/guide/basic/runtime.html)
-- [Rspack Module Federation](https://module-federation.io/guide/basic/rspack.html)
-- [Webpack Module Federation](https://module-federation.io/guide/basic/webpack.html)
-- [Next.js Module Federation](https://module-federation.io/guide/framework/nextjs.html)
+* 🎨 Module Federation Runtime
+* 🧩 Runtime Plugins System
+* 📝 Manifest
+* 🚀 Dynamic type prompt
+* 🛠️ Chrome Devtool
 
 ## 📚 Getting Started
 
 To get started with Module Federation, see the [Quick Start](https://module-federation.io/guide/start/quick-start.html).
 
 
-## 📦 Packages
-
-Some of the underlying dependencies of the past are listed here, and [enhanced](https://module-federation.io/) access is recommended for direct use.
-
-- [Node Module Federation](./packages/node)
-- [Storybook addon for Module Federation](./packages/storybook-addon)
-- [Native federation typescript](./packages/native-federation-typescript)
-- [Utilities](./packages/utilities)
-
 # 🧑‍💻 Community
 
 Come and chat with us on [Discussions](https://github.com/module-federation/universe/discussions)! The Module federation team and users are active there, and we're always looking for contributions.
```

Comment:
Should we add the discord link here?

Reply:
I think we should be able to add, was discord active before？

Reply:
Yep, its active and seeing questions pretty much every day. Here is a permanent link. https://discord.gg/n69NnT3ACV